src/common/form/DimmerField.js

```diff
@@ -27,15 +27,18 @@ const DimmerField = ({ label, name, value, toggled, onToggle, onValueChange }) =
       <div style={{ marginTop: 10, flex: 1 }}>{label}</div>
       <div style={{ whiteSpace: "nowrap", display: "flex" }}>
         <Toggle active={toggled} onClick={handleToggle} style={{ flex: 1, marginRight: 20 }} />
-        <div style={{ flex: 1, marginTop: 10 }}>
-          <ReactBootstrapSlider
-            style={{ flex: 1, paddingTop: 8 }}
+        <div style={{ flex: 1, marginTop: 0 }}>
+          <IonRange
+            style={{ flex: 1, paddingTop: 0, paddingBottom: 0, width: "210px" }}
```

Comment:
since you're doing that, can you move the css from the style attribute to css modules? See this PR as an example https://github.com/RoboDomo/robodomo-client/pull/1/commits/15622826e58de6bd29aed8cdb2c92e1db31a968f